### PR TITLE
Allow for naming individual LPC55 GPIO pins.

### PR DESCRIPTION
**Pull request by lzrd**
Used for ROT_IRQ and SP_RESET in later PRs.

### CODE REVIEW

File: drv/lpc55-gpio-api/src/lib.rs [hunk 1]
```diff
@@ -145,24 +145,64 @@ pub enum Digimode {
...
+impl From<bool> for Digimode {
+    fn from(state: bool) -> Self {
+        if state {
+            Digimode::Digital
+        } else {
+            Digimode::Analog
+        }
+    }
+}
+
```
Comment (labbott) [inline drv/lpc55-gpio-api/src/lib.rs:157]: Can you give me a pointer to where these functions actually get used in your code? I'm still not sure where they are actually needed.

**Review by labbott — COMMENTED** [review]
This does make things much nicer.

File: drv/lpc55-gpio-api/src/lib.rs (hunk 1 above)
Comment (lzrd) [reply drv/lpc55-gpio-api/src/lib.rs:157]: Bah.. I am not using them. They are leftover from some short-lived debugging.

Comment (lzrd) [reply drv/lpc55-gpio-api/src/lib.rs]: Removed

File: drv/lpc55-spi-server/src/main.rs [hunk 2]
```diff
@@ -8,6 +8,7 @@
 #![no_std]
 #![no_main]
 
+use drv_lpc55_gpio_api::Pin;
```
Comment (labbott) [inline drv/lpc55-spi-server/src/main.rs:11]: Seem to have snuck in

**Review by labbott — APPROVED** [review]
One minor cleanup but otherwise LGTM

**Pull request merged**